site/src/components/SignInForm/SignInForm.tsx

```diff
@@ -107,42 +97,44 @@ export const SignInForm: FC<SignInFormProps> = ({
     validateOnBlur: false,
     onSubmit,
   })
-  const getFieldHelpers = getFormHelpers<BuiltInAuthFormValues>(form)
+  const getFieldHelpers = getFormHelpersWithError<BuiltInAuthFormValues>(form, authError)
```

Comment:
Maybe I'm being dense :) but why do we call this method with `authError` and not `methodsError`?

Reply:
`methodsError` is not linked to the sign-in form. This method attributes form errors to specific fields in the form.